mantidproject/mantid
# Add missing algorithms to index

**To test:**
Build documentation. See that these algorithms are in the algorithm category

LoadLamp
EnggSaveSinglePeakFitResultsToHDF5
SaveGDA
ExtractQENSMembers
VesuvioPeakPrediction

Does this update require release notes?


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
